PolyEngine/Engine/Src/Engine.hpp

```diff
@@ -155,16 +158,18 @@ namespace Poly
 		inline void UpdatePhases(eUpdatePhaseOrder order)
 		{
 			HEAVY_ASSERTE(order != eUpdatePhaseOrder::_COUNT, "_COUNT enum value passed to UpdatePhases(), which is an invalid value");
-			for (auto& update : GameUpdatePhases[static_cast<int>(order)])
-				update(GetActiveScene());
+			for (auto&  update : GameUpdatePhases[static_cast<int>(order)])
+				update->OnUpdate(GetActiveScene());
 		}
 
 		/// Registers a PhaseUpdateFunction to be executed in the update.
 		/// part of a single frame in the same order as they were passed in.
 		/// @param phaseFunction - void function(Scene*)
 		/// @param order - enum eUpdatePhaseOrder value
 		/// @see eUpdatePhaseOrder
-		void RegisterUpdatePhase(const PhaseUpdateFunction& phaseFunction, eUpdatePhaseOrder order);
+		void RegisterUpdatePhase(const PhaseUpdateFunction& phaseFunction, eUpdatePhaseOrder order); //add for Isystems
```

Comment:
add [[deprecated]] attribute

Reply:
marking this deprecated causes compiler errors (warning treated as errors), as this method is still being used